Dockerfile

```diff
@@ -3,13 +3,12 @@ MAINTAINER Henning Jacobs <henning@jacobs1.de>
 
 WORKDIR /
 
-RUN pip3 install pykube
-
-WORKDIR /
-
+RUN pip3 install pipenv
```

Comment:
my main concern with pipenv in the Dockerfile is the resulting image size, see my hack to work around this here: https://github.com/hjacobs/kube-resource-report/blob/master/Dockerfile